images/tier1/divider-solid.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
+  <path fill="#565a5c" opacity=".5" d="M9 18c-.3 0-.5-.2-.5-.5V.5c0-.3.2-.5.5-.5s.5.2.5.5v17c0 .3-.2.5-.5.5z"/>
```

Comment:
Is the opacity of 0.5 intended? That's going to result in whatever background colour the divider shows up on bleeding through.